decidim-proposals/app/forms/decidim/proposals/proposal_form.rb

```diff
@@ -16,6 +16,10 @@ class ProposalForm < Decidim::Proposals::ProposalWizardCreateStepForm
       attribute :has_address, Boolean
       attribute :attachment, AttachmentForm
       attribute :suggested_hashtags, Array[String]
+      attribute :photos, Array[String]
+      attribute :add_photos, Array
+      attribute :documents, Array[String]
```

Comment:
same as before, is `Array[String]` correct?

Reply:
As I said it's an array of attachment ids. Thanks for your review!